### PR TITLE
Add missing license files and notices

### PR DESCRIPTION
**Pull request by trustin**
.. for the dependencies added since last update

### CODE REVIEW

**Comment by trustin** [comment]
@jongyeol PTAL

File: NOTICE.txt [hunk 1]
```diff
@@ -80,11 +80,26 @@ Inc:
 Dependencies
 ============
 
+This produce depends on Brave, distributed by Zipkin.io:
```
Comment (anuraaga) [inline NOTICE.txt]: product

**Comment by jongyeol** [comment]
others are LGTM.

File: NOTICE.txt (hunk 1 above)
Comment (trustin) [inline NOTICE.txt]: Hawk's eye :-) Fixed!

**Pull request merged**